config/routes.rb

```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
+  get 'home/index'
+
+  devise_for :users
+  devise_for :models
```

Comment:
I'm pretty sure we don't need `devise_for :models` here. Are you sure we need this routes?